### ISSUE TITLE
Try remote-theme plugin

### ISSUE THREAD

**Issue by dougmet** [ONSdigital/sdg-indicators#2824]
As per conversation in #2815 and PoC from @brockfanning at https://github.com/brockfanning/sdg-indicators-jekyll

This would mean that the SDG team can update all the content they want, without our commits over on the theme getting in the way. It wouldn't change anything about how a country clones the site, it just means they pick up updates when we release them (if they choose, we'd need some versioning there).

Let's have a go and see if there are any gremlins. Given our slightly complex build we might hit some issues the US don't have. Hopefully not and we can implement this quickly.

**Comment by dougmet** [comment]
Note, this is a try-it-and-report-back ticket. Not full implementation.

**Comment by shaneporter** [comment]
I've submitted #2827 for this, but here are some additional comments:

- It was straightforward enough to create a theme, but I'd say that despite the potential difficulty in bringing in upstream changes with the 'all-in-one' approach, more documentation would be required to explain how theme defaults can be overridden using this approach.
- I looked at #2826 at the same time and had some difficulties integrating the `jekyll_asset_pipeline` gem with the remote theme. Perhaps it's easily sorted, but the trial and error approach I used compounded an issue for an active developer - you have to push to the theme repo in order for the site to pick up the settings. There's a way around this I'm sure, but it adds complexity. I suppose you could use an alternate branch, which is specified in the remote theme 'client site', and then squash some commits.

**Comment by dougmet** [comment]
Awesome, thanks. I'll try this out. Not sure I understand the minification issue. Is this something that will happen at build time, or would you minify on each release of the theme?

**Comment by shaneporter** [comment]
I'm not sure how to integrate the minification at present, but the minification itself is done as part of the `jekyll_asset_pipeline` gem, so it's part of the Jekyll process.

**Comment by dougmet** [comment]
Oh OK, so it would only happen once.

**Comment by brockfanning** [comment]
I noticed that one challenge was aggregating all the JavaScript into a single sdg.js file. A possible approach is to move these files from `assets` into `_includes` so that they can be aggregated using `include`. I've tried this in a proof-of-concept, resulting in [one aggregated file](https://github.com/brockfanning/sdg-theme/tree/multilingual/assets/js) and a [folder for the separate files](https://github.com/brockfanning/sdg-theme/tree/multilingual/_includes/assets/js). One thing to keep in mind is that the separate files cannot include Liquid variables (because the double `---` at the top causes a JavaScript error).

I haven't investigated whether this helps with jekyll_asset_pipeline and minification, but it at least helps with aggregation.
